src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -3279,6 +3380,80 @@ operand immL_65535()
   interface(CONST_INTER);
 %}
 
+operand kReg()
```

Comment:
Should `kReg` operand exclude `k0` and `k1`?

Reply:
k0  - signifies all true mask.

Reply:
Yes, k0 is a special register. 
But is it even correct to pass k0 as a temp register? (`kReg` is extensively used for TEMPs.) 
According to the documentation, many instructions accepting k-registers #UD fault on k0.

Reply:
That is correct, k0 is being removed from register mask corresponding to kReg operand during register mask initializations, please refer to reg_mask_init () routine in x86_64.ad